backend/main.py

```diff
@@ -35,8 +35,12 @@ def get_config():
 @app.post("/config/")
 def add_config(config: Config):
     db = Database().get_session()
-    res = db.execute(text("INSERT INTO config (openai_api_key) VALUES (:api_key)"), {"api_key": config.apiKey})
-    db.commit()
+    try:
+        res = db.execute(text("INSERT INTO config (openai_api_key) VALUES (:api_key)"), {"api_key": config.apiKey})
+        db.commit()
+    except HTTPException as e:
```

Comment:
this would be rather a `OperationException` or something else than a `HTTPException` in my eyes.

Reply:
true, i forgot to change, will do it right now